data/json/body_parts.json

```diff
@@ -600,6 +600,103 @@
     "bmi_encumbrance_threshold": 20,
     "bmi_encumbrance_scalar": 0.15
   },
+  {
+    "id": "gastropod_foot",
+    "type": "body_part",
+    "name": "gastropod foot",
+    "accusative": { "ctxt": "bodypart_accusative", "str": "gastropod foot" },
+    "heading": "gastropod foot",
+    "heading_multiple": "gastropod feet",
+    "hp_bar_ui_text": "G. FOOT",
+    "encumbrance_text": "Running and swimming are slowed.",
+    "encumbrance_threshold": 20,
+    "main_part": "gastropod_foot",
+    "connected_to": "torso_waist",
+    "opposite_part": "gastropod_foot",
+    "limb_types": [ [ "leg_l", 0.8 ], [ "leg_r", 0.8 ], [ "foot_l", 0.5 ], [ "foot_r", 0.5 ], [ "tail", 0.2 ] ],
+    "limb_scores": [ [ "reaction", 0.3 ] ],
+    "is_vital": true,
```

Comment:
You'll die if you break your leg? I can see that, but it needs signposting in the description if so.

Reply:
Things I missed when copying from big leg.  Probably not vital?

Reply:
Probably not, but the heart is looking pretty low on most gastropod anatomy diagrams I'm looking at, depends whether you want to imply that some vital organs of Gastropod mutants are migrating, but imo the best body horror would be for all your vitals to be hidden in your shell at some later date and that "limb" replacing your torso as vital, like this diagram shows:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/16321433/98ff6015-da8c-4922-8f2e-7954d72bafe9)